tools/data-handler/src/containers/card-container.ts

```diff
@@ -51,7 +51,11 @@ export class CardContainer {
   // Finds parent
   private parentCard(cardPath: string) {
     const pathParts = cardPath.split(sep);
-    if (pathParts.at(pathParts.length - 2) === 'cardRoot') {
+    if (
```

Comment:
No need to do in this PR, but.
We should implement proper name parsing for card paths (and resources too).